docker/README.md

```
@@ -1,352 +1,548 @@
+
+**Layers:**
+
+# 2. Build and start (one command)1. **00-base** - Ubuntu 22.04, SSH, system packages, SDKMAN, Homebrew
+
+make up2. **10-languages** - Node.js, Python, Go, Rust
+
+3. **20-vscode** - VS Code Server (code-server)
+
+# 3. Access VS Code4. **30-ai-tools** - GitHub Copilot CLI, AI tool setup scripts
+
+# Open browser: http://localhost:8080
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)